### ISSUE TITLE
Pages are loading forever when asset is blocked on Windows

### ISSUE THREAD

**Issue by northys** [safing/portmaster#567]
**What happened**:

Blocked content is timeouting instead of being skipped in firefox.

~~I'm 98% sure that I saw the connection as ongoing in portmaster which was supposed to be on this video but windows is crappy bullshit and recorded only the screen I started the recording in.~~

https://user-images.githubusercontent.com/2878126/158246825-3af6f6a7-3247-4249-addc-634ef9bdb2ce.mp4

![imedia cz blocked](https://user-images.githubusercontent.com/2878126/158246903-3a5e93f7-9a08-4396-b40b-c50c65f1f232.png)

**What did you expect to happen?**:

Page loads instantly and browser is not timeouting for ~90s on blocked content.

**How did you reproduce it?**:

1) win 10
2) latest portmaster
3) open zdopravy.cz in firefox (i have same set of browser extensions on my fedora so i dont blame ublock)

**Debug Information**: https://support.safing.io/privatebin/?081940037597d922#HCuhtvUEzr6P3nZPEbYajzknsDvANnMxWiBgwDtsLXWE

**Comment by northys** [comment]
I had to reboot to Windows again and I'm still able to replicate it. It also doesn't work in Google Chrome in which I have zero plugins and I'm not even logged to my Google Account. TBH I would say I opened it for the first time today. Probably some SW installed it as affiliate :D

**Comment by northys** [comment]
Google Chrome's App Debug: https://support.safing.io/privatebin/?93fe0ae50c7bc836#8kwE8Wub513i9CEZt3RyfXGLNR2HkdPBg2K7bsWCLNSa

I can see there those domains which blocks the page rendering. It should receive instant couldn't load or something instead of timeout.

ssp.imedia.cz.
cookie-cdn.cookiepro.com.

**Comment by northys** [comment]
I noticed I did not have latest 0.8.5. Upgraded and it's still the same.

**Comment by theusaf** [comment]
Related issue: #430

**Comment by dhaavi** [comment]
Hey northys, thanks for reporting and digging into this.

I think this is a duplicate of #430 and the issue is that our Windows Kernel Extensions does not yet support "blocking" TCP connections, ie. sending a RST flag. It just drops the packets.

@northys, please check if you also feel this is a dupe of #430.

**Comment by northys** [comment]
yup, its duplicate
